clients/tabby-chat-panel/dist/react.cjs

```diff
@@ -6,18 +6,22 @@ require('@quilted/threads');
 
 function useClient(iframeRef) {
   return react.useMemo(() => {
-    if (iframeRef.current)
+    if (iframeRef.current) {
       return index.createClient(iframeRef.current);
+    }
   }, [iframeRef.current]);
 }
 function useServer(api) {
   const [isInIframe, setIsInIframe] = react.useState(false);
+  let isCreated = false;
   react.useEffect(() => {
     setIsInIframe(window.self !== window.top);
   }, []);
   return react.useMemo(() => {
-    if (isInIframe)
+    if (isInIframe && !isCreated) {
+      isCreated = true;
```

Comment:
In React, `reactStrictMode: true` will render the page twice during development, so I added a variable to prevent duplicated rendering.